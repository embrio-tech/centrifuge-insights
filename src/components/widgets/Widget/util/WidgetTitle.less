.widget-title {
  @apply text-base mb-6;
}
