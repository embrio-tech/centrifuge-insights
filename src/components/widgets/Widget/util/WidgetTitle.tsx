import React from 'react'
import './WidgetTitle.less'

interface WidgetTitleProps {
  title: string
}

export const WidgetTitle: React.FC<WidgetTitleProps> = (props) => {
  const { title } = props

  return <h3 className='widget-title'>{title}</h3>
}
