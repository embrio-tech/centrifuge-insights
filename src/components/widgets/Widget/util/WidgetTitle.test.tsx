import React from 'react'
import renderer from 'react-test-renderer'
import { WidgetTitle } from './WidgetTitle'

describe('WidgetTitle should', () => {
  test('match snapshot!', () => {
    const tree = renderer.create(<WidgetTitle title='Testing is great!' />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})

export default null
