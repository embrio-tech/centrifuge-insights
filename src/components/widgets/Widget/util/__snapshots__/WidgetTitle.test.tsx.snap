// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WidgetTitle should match snapshot! 1`] = `
<h3
  className="widget-title"
>
  Testing is great!
</h3>
`;
